<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xansible</class>
 <widget class="QMainWindow" name="xansible">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>595</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>xansible</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tab_edit_structure">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>Playbook</string>
    </property>
    <addaction name="actionNewPlaybook"/>
    <addaction name="actionOpenPlaybook"/>
   </widget>
   <widget class="QMenu" name="menuXansible">
    <property name="title">
     <string>Xansible</string>
    </property>
    <addaction name="action_apropos"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuRoles">
    <property name="title">
     <string>Roles</string>
    </property>
    <addaction name="actionNewRole"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionDocumentation"/>
   </widget>
   <widget class="QMenu" name="menuHosts">
    <property name="title">
     <string>Hosts</string>
    </property>
    <addaction name="actionNewHost"/>
   </widget>
   <widget class="QMenu" name="menuTasks">
    <property name="title">
     <string>Tasks</string>
    </property>
    <addaction name="actionNewTask"/>
   </widget>
   <addaction name="menuXansible"/>
   <addaction name="menuHosts"/>
   <addaction name="menu_Project"/>
   <addaction name="menuRoles"/>
   <addaction name="menuTasks"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionProject"/>
   <addaction name="actionNewRole"/>
   <addaction name="actionDocumentation"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionProject">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/box-solid.svg</normaloff>:/icons/ressources/box-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
  </action>
  <action name="actionNewPlaybook">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/plus-square-solid.svg</normaloff>:/icons/ressources/plus-square-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Nouveau Playbook</string>
   </property>
  </action>
  <action name="actionNewRole">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/plus-square-solid.svg</normaloff>:/icons/ressources/plus-square-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Nouveau Roles</string>
   </property>
  </action>
  <action name="action_apropos">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/adn-brands.svg</normaloff>:/icons/ressources/adn-brands.svg</iconset>
   </property>
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/power-off-solid.svg</normaloff>:/icons/ressources/power-off-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/question-circle-solid.svg</normaloff>:/icons/ressources/question-circle-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionNewHost">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/server-solid.svg</normaloff>:/icons/ressources/server-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Ajouter un serveur</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/save-solid.svg</normaloff>:/icons/ressources/save-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpenPlaybook">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/folder-open-solid.svg</normaloff>:/icons/ressources/folder-open-solid.svg</iconset>
   </property>
   <property name="text">
    <string>OpenPlaybook</string>
   </property>
  </action>
  <action name="actionNewTask">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/tasks-solid.svg</normaloff>:/icons/ressources/tasks-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Nouvelle t√¢che</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
